pisa-daemon/bpf/src/load.rs

```diff
@@ -63,7 +62,7 @@ pub enum TrafficTyp {
     SQL,
 }
 
-pub const APP_ENDPOINTS_CLASSID_PIN_PATH: &str = "/sys/fs/bpf/pisa-daemon/app-endpoints-classid";
+pub const APP_ENDPOINTS_CLASSID_PIN_PATH: &str = "/sys/fs/bpf/tc/globals";
```

Comment:
Is this should be corrected to `/sys/fs/bpf/pisa-daemon/app-endpoints-classid` ?

Reply:
This is `tc` default value when pinning is `PIN_GLOBAL_NS` or `2`.